develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,258 @@
+
+### Connect to TiDB
+
+```typescript
+import mysql from 'mysql2';
+
+let pool: mysql.Pool;
+
+function connect() {
+  pool = mysql.createPool({
+    host: process.env.TIDB_HOST, // TiDB host, for example: {gateway-region}.aws.tidbcloud.com
+    port: process.env.TIDB_PORT || 4000, // TiDB port, default: 4000
+    user: process.env.TIDB_USER, // TiDB user, for example: {prefix}.root
+    password: process.env.TIDB_PASSWORD, // TiDB password
+    database: process.env.TIDB_DATABASE || 'test', // TiDB database name, default: test
+    ssl: {
+      minVersion: 'TLSv1.2',
+      rejectUnauthorized: true,
+    },
+    connectionLimit: 1, // Setting connectionLimit to "1" in a serverless function environment optimizes resource usage, reduces costs, ensures connection stability, and enables seamless scalability.
+    maxIdle: 1, // max idle connections, the default value is the same as `connectionLimit`
+    enableKeepAlive: true
+  });
+}
+
+export async function handler(event: any) {
+  if (!pool) {
+    connect();
+  }
+  const results = await pool.execute('SELECT "Hello World"');
+  return {
+    statusCode: 200,
+    body: JSON.stringify({ results }),
+  };
+}
+```
+
+### Insert data
+
+```typescript
+const player = ['1', 10, 500];
+await pool.execute(
+  `INSERT INTO player (id, goods, coins) VALUES (${player[0]}, ${player[1]}, ${player[2]})`
+);
+```
+
+Refer to [Insert data](/develop/dev-guide-insert-data.md) for more information.
+
+### Query data
+
+```typescript
+const [rows] = await pool.execute('SELECT count(*) AS cnt FROM player');
+console.log(rows[0]['cnt']);
+```
+
+Refer to [Query data](/develop/dev-guide-get-data-from-single-table.md) for more information.
+
+### Update data
+
+```typescript
+const player = ['1', 10, 500];
+await pool.execute(
+  `UPDATE player SET goods = goods + ${play[0]}, coins = coins + ${play[1]} WHERE id = ${play[2]}`
+);
+```
+
+Refer to [Update data](/develop/dev-guide-update-data.md) for more information.
+
+### Delete data
+
+```typescript
+await pool.execute('DELETE FROM player WHERE id = 1');
+```
+
+Refer to [Delete data](/develop/dev-guide-delete-data.md) for more information.
+
+## Useful notes
+
+### Using driver or ORM framework?
+
+The Node.js driver provides low-level access to the database, but it requires the developers to:
+
+- Manually establish and release database connections.
+- Manually manage database transactions.
+- Manually map data rows to data objects.
+
+Unless you need to write complex SQL statements, it is recommended to use [ORM](https://en.wikipedia.org/w/index.php?title=Object-relational_mapping) framework for development, such as [Sequelize](https://sequelize.org/), [Prisma](https://www.prisma.io/), and [TypeORM](https://typeorm.io/). It can help you:
+
+- Reduce [boilerplate code](https://en.wikipedia.org/wiki/Boilerplate_code) for managing connections and transactions.
+- Manipulate data with data objects instead of a number of SQL statements.
```

Comment:
Provide some more useful notes about lambda functions/next.js instead of repeating `Useful notes` from the Python tutorial